static/talawa/static-assets/styles.css

```diff
@@ -84,7 +84,7 @@ main {
   flex: 0 1 230px;
   order: 1;
   overflow-y: scroll;
-  padding: 20px 0 15px 30px;
```

Comment:
- Changes to styles.css are not scope of the PR. Only fix the dates. Please revert.
- Changes to index.html are not scope of the PR. Only fix the dates. Please revert.

Reply:
@palisadoes Please disregard this PR, it was a mistake